docs/deploy-azure/toc.yml

```diff
@@ -32,6 +32,8 @@
     href: /vsts/build-release/actions/define-multistage-release-process?toc=/vsts/deploy-azure/toc.json&bc=/vsts/deploy-azure/breadcrumb/toc.json
   - name: CI builds for Git in VSTS
     href: /vsts/build-release/actions/ci-build-git?toc=/vsts/deploy-azure/toc.json&bc=/vsts/deploy-azure/breadcrumb/toc.json
+  - name: Deploy apps to Azure with VSTS and CI/CD
```

Comment:
I'd suggest Deploy Web Apps to Azure with VSTS and CI/CD as the name, since it is very much about Web Apps.